project_electron/orgs/models.py

```diff
@@ -242,10 +244,16 @@ def bag_or_failed_name(self):
     def gen_identifier(self,fname,org,date,time):
         return "{}{}{}{}".format(fname,org,date,time)
 
-    def get_errors(self):
+    def get_error_codes(self):
         if self.bag_it_valid:
             return ''
-        return [b.code.code_desc for b in BAGLog.objects.filter(archive=self).exclude(code__code_short='ASAVE')]
+        return [b.code.code_desc for b in self.get_errors()]
+
+    def get_errors(self):
+        if self.bag_it_valid:
+            return None
+        return [b for b in BAGLog.objects.filter(archive=self).exclude(code__code_short='ASAVE')]
```

Comment:
The `PBAG` "Passed Bag Validation" messages are appearing in error logs, which seems semantically wrong. I think those messages should also be excluded from the error logs.

Reply:
Ahh yeah.. lol.. Thanks